src/Dynatrace.OpenTelemetry.Exporter.Metrics.Tests/DynatraceMetricsExporterTests.cs

```diff
@@ -663,35 +636,42 @@ public async Task Export_ObservableDoubleCounter()
 
 			var sut = new DynatraceMetricsExporter(null, null, new HttpClient(mockMessageHandler.Object));
 
-			// Delta is already the preferred temporality for our exporter
+			// Delta is already the preferred/temporality temporality for our exporter
 			var metricReader = new TestMetricReader(sut);
 			using var provider = Sdk.CreateMeterProviderBuilder()
 				.AddMeter(meter.Name)
 				.AddReader(metricReader)
 				.Build();
 
+			var i = 1;
 			var observableCounter = meter.CreateObservableCounter("double_obs_counter", () =>
 			{
 				return new List<Measurement<double>>()
 				{
-					new Measurement<double>(10.3, _attributes)
+					new(i++ * 10.3, _attributes)
 				};
 			});
 
-			// Act - Reader will call our exporter
+			// Perform two exports to ensure deltas are exported correctly
 			metricReader.Collect();
+			await AssertLines(metricReader, 10.3);
 
-			// Assert
-			var exportedMetrics = metricReader.GetExportedMetrics();
-			var point = MetricTest.FromMetricPoints(exportedMetrics.First().GetMetricPoints()).First();
+			metricReader.Collect();
+			await AssertLines(metricReader, 10.3);
```

Comment:
This means that the second time the total value would be 20.6, right?

Reply:
Yes, but because it exports deltas, the SDK gives the correct value of  `10.3`